File: Python/program/electronic.py
# electronic.py
class Electronic:
    def __init__(self, brand, year):
        self._brand = brand
        self._year = year
    
    def get_brand(self):
        return self._brand
    
    def set_brand(self, brand):
        self._brand = brand
    
    def get_year(self):
        return self._year
    
    def set_year(self, year):
        self._year = year
    
    def display_info(self):
        print(f"Brand: {self._brand}, Year: {self._year}")

File: CPP/program/Electronic.cpp
#pragma once
#include <iostream>
#include <string>

class Electronic {
protected:
    std::string brand;
    int year;
public:
    Electronic(std::string brand, int year) : brand(brand), year(year) {}
    virtual ~Electronic() { std::cout << "Electronic device destroyed\n"; }
    std::string getBrand() const { return brand; }
    void setBrand(std::string brand) { this->brand = brand; }
    int getYear() const { return year; }
    void setYear(int year) { this->year = year; }
    virtual void displayInfo() const {
        std::cout << "Brand: " << brand << ", Year: " << year << std::endl;
    }
};